primitive-types/impls/rlp/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "impl-rlp"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
I think this is technically a breaking change, since `rlp::Encodable` from 0.4 is incompatible with 0.3 one according to semver, right?

Reply:
Hmm, the trait `Encodable` didn't change. Only which types in `rlp` implements it, and those changes are not relevant to `impl-rlp` (because it impls `Encodable` for the same types) so I think this one is fine to consider a patch release. Or else, where do we stop? Breaking release for `primitive-types` too? It sort of snowballs...

Reply:
I meant this: https://github.com/paritytech/parity-common/compare/ao-snowballing-is-unfortunate